starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,64 @@
+
+ * {
+     font-family: Open Sans;
+ }
+ 
+ .header {
+    background: url(/Users/jonathansanchez/Desktop/Ironhack/IndependentExercises/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg) 0 0 no-repeat;
```

Comment:
We don't use the path to the image that is only available on your local machine. Instead we would use something like background:url("images/someImage.jpg")